src/SearchBtn.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+const SearchBtn = () => {
+    return (
+        <button className="btn btn-primary">Search IDs</button>   
```

Comment:
How could we make this component more reusable? Right now if we tried to use this component elsewhere in the app it would still say "Search IDs". How could we change it so that it can be used in multiple places?